src/GraphWebhooks-Core/Views/Shared/_LoginPartial.cshtml

```diff
@@ -1,15 +1,17 @@
-﻿@using System.Security.Principal
```

Comment:
the change in this file is no longer necessary with the new version of Microsoft.Identity.Web